infrastructure/router/router.go

```diff
@@ -0,0 +1,32 @@
+package router
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/ErickRodriguezWize/academy-go-q42021/interface/controller"
+	"github.com/ErickRodriguezWize/academy-go-q42021/service"
+
+	"github.com/gorilla/mux"
+)
+
+func NewRouter() {
+	r := mux.NewRouter()
+
+	pokemonHandler := controller.NewPokemonController()
+
+	//Pokemon Endpoints
+	r.HandleFunc("/pokemons", pokemonHandler.GetAllPokemons).Methods("GET")
+	r.HandleFunc("/pokemons/{id}", pokemonHandler.GetPokemon).Methods("GET")
+
+	//Server setup
+	srv := &http.Server{
+		Handler: r,
+		Addr: service.GetEnvVariable("IP_ADDRESS") + ":" + service.GetEnvVariable("PORT"),
+	}
+
+	log.Println("Server Started at", srv.Addr)
+
+	log.Fatal(srv.ListenAndServe())
+
+}
```

Comment:
Please leave all your files with an empty line at the end of them.

This is a common standard that is intended to be followed in Go.